src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1617,6 +1617,9 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public string ToUpper() { return default(string); }
         public string ToUpperInvariant() { return default(string); }
         public string Trim() { return default(string); }
+        public string Trim(char trimChars) { return default(string); }
```

Comment:
Were these a part of an API review? If so can you please link to it and if not these need to be reviewed before adding them.